app/services/taxonomy/build_taxon.rb

```diff
@@ -3,6 +3,7 @@ class BuildTaxon
     attr_reader :content_id
 
     class TaxonNotFoundError < StandardError; end
+    class ContentTypeError < StandardError; end
```

Comment:
`DocumentTypeError` might be more fitting here, as the field being checked is the `document_type`?

Reply:
Updated in https://github.com/alphagov/content-tagger/pull/568/commits/b3a255810bd5e4b18ed85d1ade3b2a0b64059b30

Reply:
Great :)